mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java

```diff
@@ -171,4 +171,5 @@
     public static final int NOTIFICATION_ID_BIG_IMAGE = 101;
 
 
+    public static final String SELECT_A_SIM = "Select a Sim";
```

Comment:
why is a toast message value present in `Constants`.
This should be in `strings.xml` and then use `context.getString()`

Reply:
@naman14 updated.